classes/migration/upgrade/v3_6_0/I857_ContributorRolesTypes.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/v3_6_0/I857_ContributorRolesTypes.php
+ *
+ * Copyright (c) 2025 Simon Fraser University
+ * Copyright (c) 2025 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class I857_ContributorRolesTypes
+ *
+ * @brief OMP specific actions related to contributor roles and types
+ */
+
+namespace APP\migration\upgrade\v3_6_0;
+
+use Illuminate\Support\Facades\DB;
+use PKP\install\DowngradeNotSupportedException;
+use PKP\migration\Migration;
+
+class I857_ContributorRolesTypes extends Migration
```

Comment:
Maybe to extend the PKP class and then first run parent::up(), and then only use this one in upgrade.xml